fem/libceed/ceed.cpp

```diff
@@ -35,6 +35,9 @@ extern Ceed ceed;
 
 std::string ceed_path;
 
+extern khash_t(basis) *ceed_basis_from_fes;
+extern khash_t(restr) *ceed_restr_from_fes;
```

Comment:
Hmmm... what these? Where are they defined?

Reply:
These functions are part of Khash. Khash is a header only hashing library that we have been using in libCEED. These functions are defined in `khash.h` and `ceed-hash.h` in libCEED's include dir.

If MFEM's native hash tables can map to arbitrary types instead of only integers (this did not look to be the case), I can use that instead in this PR.